### ISSUE TITLE
debug indicators eg: trigger, start, end does not display

### ISSUE THREAD

**Issue by muyaszed** [janpaepke/ScrollMagic#224]
Hi guys, as per the subject explains, got any idea why?
```js
// init controller
  var controller = new ScrollMagic();

  // build tween
  var tween = TweenMax.to("#logo", 1, {scale: 0.35});

  // build scene
  var scene = new ScrollScene({triggerElement: "#trigger1"})
                  .setTween(tween)
                  .addTo(controller);

  // show indicators (requires debug extension)
  scene.addIndicators();
```
Just started trying this plugin and copy and paste some examples here.
By the way im using foundation framework, and the scripts are set on a separate file.

Thanks

**Comment by justinmarsan** [comment]
Are you loading the separate debug file ?

If so, check the before the body closing tag for the `.ScrollSceneIndicators` if they're not there and you do have the debug lib loaded, something is wrong and I don't really know what, if they are indeed here but don't appear on screen, I'd bet on a z-index issue or something like that...

**Comment by muyaszed** [comment]
Hi Justin Thanks for the reply, do you mean the jquery file? if yes this is what i have:

$otherhook = $(".ScrollSceneIndicators .hook[data-hook=\""+ hookPos +"\"][data-direction="+direction+"]:visible").not($triggerHook);
			if ($otherhook.length > 0 && $otherhook.data("parent") == cParams.container) {
				$triggerHook.hide();
			} else {
				$triggerHook.show();
				var flip = hookPos > cParams.size*0.8; // put name above line?	
				if (cParams.vertical) {
					// triggerHook
					$triggerHook.css({
						top: flip ? hookPos - $triggerHook.height() - 2 : hookPos,
						left: (cParams.isDocument ? cParams.container.width() : parentOffset.left + cParams.container.width() - $(document).scrollLeft()) - 70 - options.indent,
						width: 40,
						height: "auto",
						padding: "0 8px 2px 8px",
						"border-top": flip ? "none" : "1px solid blue",
						"border-bottom": flip ? "1px solid blue" : "none",
						"border-left": "none",
						"border-right": "none"
					});

and more.......

Where can i found out more about the z-index.

Thanks

**Comment by justinmarsan** [comment]
Hey.

Yes, looks like it.

Did you look at your DOM, in a browser inspector, and see the `.ScrollSceneIndicators` ? You should see elements dynamically created just before the closing body tag of your page. Do you ?

**Comment by justinmarsan** [comment]
You can also try and add the following code to your CSS and see if it fixes things : 

    .ScrollSceneIndicators { z-index: 99999 !important; }

**Comment by muyaszed** [comment]
thats a great fix but partially. Now the texts appeared but not moving. Thanks anyway

**Comment by justinmarsan** [comment]
Ok, there might be some issue around I'm not aware of, I'll leave it to Jan to take a look at this issue then, unless you have a live example or something, I don't think I can help more here...

**Comment by muyaszed** [comment]
Thanks again. meanwhile i would try and explore it myself too. Unfortunately im running on a local server to give you guy a peek.

**Comment by janpaepke** [comment]
@justinmarsan thanks for your help! :)

First I'd like to note that there is an option to supply the z-index to he indicators by calling them like this:
```js
scene.addIndicators({zindex: 999});
```
Note: This will not be necessary anymore for ScrollMagic 2.0 (in alpha stadium right now).

The reason this was introduced in the first place is that a layer on top of the content (and it's on top, when zindex is high) won't pass on mousewheel events, triggered when the mouse is on top of them.
So when pushing up the zindex your (@muyaszed) site should work, but only if you use the scrollbar.

In ScrollMagic 2.0 this has been resolved by using the CSS line `pointer-events: none`, which prevents the mousewheel events to be captured by the indicators.
This is why there won't be any more zindex problems in 2.0.
This css fix only works in IE11+, which is okay considering, that you hopefully are not developing in IE9 or 10 and the indicators are purely for the development phase.

So in short:
To fix your problem right now, try this in addition to the z-index definition:
```css
.ScrollSceneIndicators { pointer-events: none; }
```

**Comment by muyaszed** [comment]
Hi Jan,

Thanks for the reply, Ive check the element originally the pointer-events is already set to none.

FYI when I apply 
    .ScrollSceneIndicators { z-index: 99999 !important; } inside css

Both Hook and start indicator are showing but start indicator is not moving and when i apply
    scene.addIndicators({zindex: 999}); inside js

Only the start indicator showed up and still not moving, either way any tweein , pin etc effect will works

**Comment by janpaepke** [comment]
Do you use a ScrollContainer? If so you might also need to define the `parent` option for the indicators...

**Comment by muyaszed** [comment]
Hi Guys,

just found out that, some css element imported from another plugin im using causing all these problems. Feeling better now. But still having problem integrating both. Anyway, thanks alot to everybody in helping me. thanks again.

**Comment by janpaepke** [comment]
Well the debug extension is just for development phase. So you might not need to integrate both. :)
I'd be curious to know which plugin it is though?
And if you  have the time you can test if these problems still exist if you use [ScrollMagic 2.0 alpha](https://github.com/janpaepke/ScrollMagic/releases/tag/v2.0.0-alpha).

**Comment by muyaszed** [comment]
Finally, a content wrapper div which have 100% height is the culprit. I'm using an off canvas menu plugin from Codrops and sure no prob if i have time i will try to use the 2.0.
